src/pages/inputMethod.tsx

```diff
@@ -18,6 +16,10 @@ type InputMethodProps = {
     errors: ErrorInfo[];
 };
 
+export const errorsExist = (
```

Comment:
there is a typechecking file where you can add this but the method will have to be more explicit as we have different ones in there